3drepo/3drepo.io
# ISSUE 4991 - Send io-activity logs to Elastic

This fixes #4991 

#### Description
- NET logs get sent to Elastic if configured

#### Test cases
- Use 3drepo.io with Elastic setup
- Check Elastic for `io-activity` index and records
